firebase-firestore/src/main/java/com/google/firebase/firestore/AggregateQuery.java

```diff
@@ -70,14 +69,35 @@ public Task<AggregateQuerySnapshot> get(@NonNull AggregateSource source) {
     return tcs.getTask();
   }
 
+  /**
+   * Compares this object with the given object for equality.
+   *
+   * <p>This object is considered "equal" to the other object if and only if all of the following
```

Comment:
Please double-check whether closing </p> and </li> tags are required here and a few lines below, and in corresponding docstrings in other source files.

Reply:
I originally had the closings tags in there, but the code formatter actually removed them!